src/panels/lovelace/hui-root.ts

```diff
@@ -0,0 +1,583 @@
+                      }
+                      <paper-item @click="${this._handleUnusedEntities}"
+                        >Unused entities</paper-item
+                      >
+                      <paper-item @click="${this._editModeEnable}"
+                        >${
+                          this.localize("ui.panel.lovelace.editor.configure_ui")
+                        }</paper-item
+                      >
+                      ${
+                        this._storageMode
+                          ? html`
+                              <paper-item
+                                @click="${this.lovelace!.enableFullEditMode}"
+                                >Raw config editor</paper-item
+                              >
+                            `
+                          : ""
+                      }
+                      <paper-item @click="${this._handleHelp}">Help</paper-item>
+                    </paper-listbox>
+                  </paper-menu-button>
+                </app-toolbar>
+              `
+        }
+
+        ${
+          this.lovelace!.config.views.length > 1 || this._editMode
+            ? html`
+                <div sticky>
+                  <paper-tabs
+                    scrollable
+                    .selected="${this._curView}"
+                    @iron-activate="${this._handleViewSelected}"
+                  >
+                    ${
+                      this.lovelace!.config.views.map(
+                        (view) => html`
+                          <paper-tab>
+                            ${
+                              view.icon
+                                ? html`
+                                    <ha-icon
+                                      title="${view.title}"
+                                      .icon="${view.icon}"
+                                    ></ha-icon>
+                                  `
+                                : view.title || "Unnamed view"
+                            }
+                            ${
+                              this._editMode
+                                ? html`
+                                    <ha-icon
+                                      class="edit-view-icon"
+                                      @click="${this._editView}"
+                                      icon="hass:pencil"
+                                    ></ha-icon>
+                                  `
+                                : ""
+                            }
+                          </paper-tab>
+                        `
+                      )
+                    }
+                    ${
+                      this._editMode
+                        ? html`
+                            <paper-button
+                              id="add-view"
+                              @click="${this._addView}"
+                            >
+                              <ha-icon
+                                title="${
```

Comment:
`.title` I could be wrong here. But since its dynamic property is better right

Reply:
It was set in Polymer like `title$` which meant set it as attribute. That makes sense so the browser will show the tooltip